docs/api_docs/kubeblocks-for-milvus/manage-milvus.md

```diff
@@ -0,0 +1,620 @@
+---
+title: Manage Milvus with KubeBlocks
+description: How to manage Milvus on KubeBlocks
+keywords: [milvus, control plane]
+sidebar_position: 1
+sidebar_label: Manage Milvus with KubeBlocks
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Manage Milvus with KubeBlocks
+
+The popularity of generative AI (Generative AI) has aroused widespread attention and completely ignited the vector database (Vector Database) market.
+
+Milvus is a highly flexible, reliable, and blazing-fast cloud-native, open-source vector database. It powers embedding similarity search and AI applications and strives to make vector databases accessible to every organization. Milvus can store, index, and manage a billion+ embedding vectors generated by deep neural networks and other machine learning (ML) models.
+
+## Before you start
+
+* [Install KubeBlocks](./../installation/install-kubeblocks.md).
+* View all the database types and versions available for creating a cluster.
+  
+  Make sure the `milvus` cluster definition is installed. If the cluster definition is not available, refer to [this doc](./../overview/supported-addons.md#install-addons) to enable it first.
+
+  ```bash
+  kubectl get clusterdefinition milvus
+  >
+  NAME         TOPOLOGIES   SERVICEREFS   STATUS      AGE
+  milvus                                  Available   30m
+  ```
+
+  View all available versions for creating a cluster.
+
+  ```bash
+  kubectl get clusterversions -l clusterdefinition.kubeblocks.io/name=milvus
+  ```
+
+* To keep things isolated, create a separate namespace called `demo` throughout this tutorial.
+
+  ```bash
+  kubectl create namespace demo
+  ```
+
+## Create a cluster
+
+KubeBlocks implements a `Cluster` CRD to define a cluster. Here is an example of creating a Milvus cluster.
+
+```yaml
+cat <<EOF | kubectl apply -f -
+apiVersion: apps.kubeblocks.io/v1alpha1
+kind: Cluster
+metadata:
+  name: mycluster
+  namespace: demo
+spec:
+  affinity:
+    podAntiAffinity: Preferred
+    tenancy: SharedNode
```

Comment:
tenancy: SharedNode is deprecated, better to remove

Reply:
> tenancy: SharedNode is deprecated, better to remove

Fixed.